Fixed broken references for compiling and contactus in README.md

divark opened the pull request:
Also added Mac OS X compiling instructions.

divark committed:
Fixed broken references to compiling and contact us. Also added Mac OS instructions for compiling.

obiwankennedy reviewed (commented):
Thank for the updates. I'm fine with all your changes.

But, As **Rolisteam** is now under KDE projects, I should not use github anymore but the gitlab from KDE for merge requests. 
The KDE gitlab has some rules about commit. 
In our case, commit author should have defined with two words (First name and last name). 
I suspect your commit to be labeled with your nickname "divark".  So If I merge your commit to github, I won't be able to commit it into KDE  gitlab.
I will have to edit your name in the commit. I can do it but it is a bit painful. 
If you don't want to show your real name. You can set Divark Divark as author name it will be fine.

Let me know if you can take care of that on your side. Otherwise I will do it in few days.

Please, let me know.

divark commented:
Hello!

I am fine with it showing my real name. Looking into it, my author name is already set to my real name according to git log:
```
commit 68692c10f19c72dadc97f42ecfd42772e0320e24 (HEAD -> master, origin/master, origin/HEAD)
Author: Tyler Schmidt <...>
Date:   Thu Jan 23 22:20:04 2020 -0800

    Fixed broken references to compiling and contact us. Also added Mac OS instructions for compiling.

commit 837ece232fe134fcc43e1db3ad0b17c128f3a402
Author: Renaud G <renaud@rolisteam.org>
Date:   Mon Dec 23 23:41:19 2019 +0100

    Fix diceshortcut roll.
...
```

Though, GitHub only shows the username per commit. I don't think there's much I can do on my end because of this. If you'd like, I could open a merge request on the KDE gitlab, and close this one if it makes it any easier.

divark committed:
Fixed reference to License
